src/api/queries/videos.graphql

```diff
@@ -94,6 +94,23 @@ query GetVideos($offset: Int, $limit: Int, $where: VideoWhereInput, $orderBy: Vi
   }
 }
 
+query GetChannelVideosPreview($limit: Int = 10, $channelId: ID) {
+  videos(
+    offset: 0
+    limit: $limit
+    where: {
+      channelId_eq: $channelId
+      isPublic_eq: true
+      isCensored_eq: false
+      thumbnailPhotoAvailability_eq: ACCEPTED
+      mediaAvailability_eq: ACCEPTED
+    }
+    orderBy: [createdAt_DESC]
+  ) {
+    ...VideoFields
+  }
+}
+
```

Comment:
I'm not sure if we actually need this. We could just use regular `GetVideos` here, we are creating a separate hook anyway for this.

Reply:
Agreed. I thought separate query name would separate the entries in cache but it actually doesn't care so I guess it doesn't make sense to introduce a separate one. Removed now